O-Osahon/World-News
# #168503264 Setup React Environment

### What does this PR do?
- Sets up react environment 

### Description of tasks to be completed
- Setup React using Webpack and Babel

### How can this be manually tested?
- Clone this repo and checkout this branch using the following command:

```
git branch -r     

git checkout -b ch-setup-react-168503264  origin/ch-setup-react-168503264
```
- Run `npm install` to install dependencies
- Run `npm start`
- The application should now be running on `localhost:8000`
